lib/features/email/presentation/widgets/html_attachment_previewer.dart

```diff
@@ -0,0 +1,96 @@
+
+import 'package:core/presentation/utils/responsive_utils.dart';
+import 'package:core/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart';
+import 'package:core/presentation/views/responsive/responsive_widget.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:pointer_interceptor/pointer_interceptor.dart';
+import 'package:tmail_ui_user/features/email/presentation/widgets/pdf_viewer/top_bar_attachment_viewer.dart';
+import 'package:tmail_ui_user/main/utils/app_utils.dart';
+
+class HtmlAttachmentPreviewer extends StatelessWidget {
+  const HtmlAttachmentPreviewer({
+    super.key,
+    required this.htmlContent,
+    required this.title,
+    required this.mailToClicked,
+    required this.downloadAttachmentClicked,
+  });
+
+  final String title;
+  final String htmlContent;
+  final void Function(Uri? mailToUri) mailToClicked;
+  final VoidCallback downloadAttachmentClicked;
+
+  static const double _verticalMargin = 16;
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      children: [
+        TopBarAttachmentViewer(
+          title: title,
+          closeAction: Get.back,
+          downloadAction: downloadAttachmentClicked,
+        ),
+        Expanded(
+          child: LayoutBuilder(
+            builder: (context, constraints) {
+              return SingleChildScrollView(
+                child: GestureDetector(
+                  onTap: Get.back,
+                  child: PointerInterceptor(
+                    child: ColoredBox(
+                      color: Colors.transparent,
+                      child: Center(
+                        child: Container(
+                          margin: const EdgeInsets.symmetric(vertical: _verticalMargin),
+                          color: Colors.white,
+                          child: ResponsiveWidget(
+                            responsiveUtils: ResponsiveUtils(),
```

Comment:
Please use singleton for `ResponsiveUtils `

Reply:
Done